docs/source/user-guide/nlp/supported_tasks.rst

```diff
@@ -0,0 +1,100 @@
+.. _supported_tasks:
+
+===================================
+Working with Labels and Predictions
+===================================
+
+Some checks, mainly the ones related to model evaluation, require labels and model predictions in order to run.
+In the deepchecks nlp package, predictions are passed into the suite / check ``run`` method as pre-computed
+predictions only (passing a fitted model is currently not supported).
+
+
+.. _supported_task__types:
+
+Supported Task Types
+====================
+
+Deepchecks currently supports two NLP task types:
+
+* **Text Classification**: Text classification is any NLP task in which a whole body of text (ranging from a sentence
+  to a document) is assigned a class (in the binary/multiclass case) or a certain set of classes (in the multilateral
+  case). In both the binary, the multiclass and the multilabel case the class "belongs" / "classifies" the whole text
+  sample. Examples for such tasks are Sentiment Analysis, Topic Extraction, detecting harmful content and more.
+* **Token Classification**: Token Classification is any NLP task in which each word (or to be more accurate, token) in
+  the text sample is assigned a class of it's own. In many cases most tokens will belong to a "background" class,
+  allowing the model to focus on the interesting tokens. Examples for such tasks are Named Entity Recognition, Part
+  of Speach annotation (in which all tokens have a non-background class).
+
+.. _supported_labels__predictions_format:
+
+Supported Labels and Predictions Format
```

Comment:
To be honest when I read this section - it makes me feel that it's more convenient for the hierarchy to be:

Task Type 1
===========
Label Format
--------------

Prediction Format
---------------------

Task Type 2
==========
Label Format
--------------

Prediction Format
-------------------


and not the current (label, prediction as outer, task type as inner), as this is more natural in the user's flow.... WDYT?
If so - we can open issues (for tabular, nlp, etc...) and tag them as "first good issues".
Anyway wouldn't consider it as urgent, but could be helpful....

Reply:
Done